client/src/pages/Register/index.tsx

```diff
@@ -96,17 +104,21 @@ class App extends React.Component<Props, RegistrationState> {
       });
 
       this.setState({
-        status: "submitted",
+        status: "submitting",
```

Comment:
I don't understand this change - the POST request is finished at this point, so isn't "submitted" the correct status?

Reply:
I think we can change this, but if you look at the `render` method you can see that if the status is "submitted" then it will render the confirmation template.  We can try and rejigger this, but I think it would involve some renaming and perhaps rethinking since we've now got both `status` and `step` which represent different states of the app which probably need to be married and normalized.

Reply:
I think it would make sense to rejigger it - my thought the existing `status` values would still make sense, and `step` could be "registration", "payment" or "confirmation". We'd render the confirmation template if `step` is "confirmation". Does that make any sense?